### ISSUE TITLE
More than one repository

### ISSUE THREAD

**Issue by hackmonker** [azinchen/duplicacy#180]
How should I go about backing up two different repositories to different storages? rn I can backup /opt to my NAS but I also want to backup /mnt to drive. normally i can simply init the two repositories with their respective storage urls and then cd to whichever i want and backup. but how to accomplish the same here?

**Comment by azinchen** [comment]
Multiple backup storage is not supported in `azinchen/duplicacity`. As a workaround you can create two different containers.

**Comment by hackmonker** [comment]
ok thanks. A bit of doubt. If I want to backup /opt folder, should I simply use -v /opt:/data ? I mean I only worked with cli before so not sure what is the current working directory for the docker and what it will use as repository

**Comment by azinchen** [comment]
Yes, you can simply mount `/opt` folder to `/data` of `azinchen/duplicacity`. Repository folder is created during each create of container, and if you would like to store this folder for avoiding reinitialization, manual tuning and define pre/post scripts, you should mount `/config` folder of `azinchen/duplicacity` as well.

**Comment by hackmonker** [comment]
Can you tell where the .duplicacy folder is located? I cant find any .duplicacy folder in /data even though that should be source I believe. there is one in config but its a file and not a directory

**Comment by hackmonker** [comment]
i wanted to define a few post scripts but I need to use .duplicacy/scripts/ folder but cant find the .duplicacy folder

**Comment by hackmonker** [comment]
how do we add the gcd token in such a docker?
